.github/actions/get-maintained-releases/action.yml

```diff
@@ -0,0 +1,45 @@
+name: "Get list of maintained releases"
+description: >
+  Goes through the list of release branches in the chisel-releases repository
+  and outputs a list of the currently maintained releases.
+
+outputs:
+  maintained-releases:
+    description: "List of currently maintained releases"
+    value: ${{ steps.get-maintained-releases.outputs.maintained-releases }}
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y distro-info
+
+    # We could rely on the chisel.yaml "maintained" field, but that would
+    # require checking out each releases parsing the file.
+    # Instead, we can use the distro-info tool, because in the end, what we
```

Comment:
> what we care about is whether the release is maintained or not, and not its current maintenance status.

i don't understand this sentence. feels tautological. surely whether a release is maintained of not *is* is maintenance status?

Reply:
our concern here is boolean, i.e. it reached EOL or not.

The `maintenance` field gives you much more info that than. It tells you **where** in the lifecycle of a release you are (e.g. devel, esm, etc.), and this scripts doesn't care about that